py/client/build.gradle

```diff
@@ -22,40 +23,13 @@ dependencies {
     python project(path: ':proto:proto-backplane-grpc', configuration: 'python')
 }
 
-def generatedProtoDir = layout.buildDirectory.dir('generatedProto')
-
-def updatePyNamespaces = tasks.register('updatePyNamespaces', Sync) {
-    from(configurations.python) {
-        // change directory from deephaven to pydeephaven
-        eachFile { it.path = 'pydeephaven/' + it.path.substring('deephaven/'.length()) }
-        // rewrite specific imports/namespaces from deephaven to pydeephaven
-        filter { contents ->
-            def scanner = new Scanner(contents)
-            def lines = []
-            while (scanner.hasNextLine()) {
-                def line = scanner.nextLine();
-                if (line.trim().startsWith('\'__module__\' :')) {
-                    line = line.replace('\'deephaven.', '\'pydeephaven.')
-                } else if (line.startsWith('from deephaven.proto')) {
-                    line = line.replace('from deephaven.proto', 'from pydeephaven.proto')
-                }
-                lines.add(line)
-            }
-            return lines.join(System.lineSeparator());
-        }
-    }
-
-    into generatedProtoDir
-}
-
 def compare = tasks.register('compareProtobuf', DiffTask) {
-    dependsOn updatePyNamespaces
-
-    expectedContents.set generatedProtoDir.get().dir('pydeephaven/proto')
+    expectedContents.set configurations.python
     actualContents {
-        directory layout.projectDirectory.dir('pydeephaven/proto')
+        directory layout.projectDirectory
         ignore {
-            it.include '__init__.py'
+            it.exclude 'deephaven_core/proto/**'
+//            it.include '__init__.py'
```

Comment:
Useful to leave as a comment?

Reply:
Ah ty - we decided we don't need this, so I'll remove it.